### PR TITLE
WFLY-9911 Move dependency managment under separate bom

### PR DESCRIPTION
**Pull request by ctomc**
https://issues.jboss.org/browse/WFLY-9911

### CODE REVIEW

**Comment by stuartwdouglas** [comment]
Failures look related

File: testsuite/integration/pom.xml [hunk 1]
```diff
@@ -385,7 +385,7 @@
             <artifactId>wildfly-transaction-client</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
+       <!-- <dependency>
```
Comment (stuartwdouglas) [inline testsuite/integration/pom.xml]: If things are no longer needed they should be removed and not commented out.

**Review by stuartwdouglas — CHANGES_REQUESTED** [review]
This needs a rebase and for the failures to be fixed

**Comment by kabir** [comment]
@ctomc needs a rebase

**Review by stuartwdouglas — APPROVED** [review]

**Pull request merged**